broken_record.gemspec

```diff
@@ -19,9 +19,11 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_development_dependency 'bundler', '~> 1.3'
+  spec.add_development_dependency 'coveralls', '~> 0.8.1'
```

Comment:
can we split the coverage stuff into another PR and send it to code climate instead?

Reply:
Sure